src/i18n/strings/en_EN.json

```diff
@@ -2221,6 +2221,8 @@
             "no_aliases_space": "This space has no local addresses",
             "other_section": "Other",
             "publish_toggle": "Publish this room to the public in %(domain)s's room directory?",
+            "error_publishing": "There was an error publishing this room",
+            "error_publishing_detail": "Not allowed to publish room",
```

Comment:
This is what the error is in this case for matrix.org, but there could be other errors, and unfortunately the error code in this case is just M_UNKNOWN, so we can't really be any more specific. I'd suggest using, "Unable to publish room" as the title and, "There was an error publishing this room" as the body, perhaps?

Reply:
done